dear_petition/petition/export/tests/test_mapper.py

```diff
@@ -29,3 +35,66 @@ def test_map_petition__district(petition):
     mapper.map_petition(data, petition)
     assert data["District"] == "Yes"
     assert data["Superior"] == ""
+
+
+# map_petitioner tests
+def test_map_petitioner__name(petition):
+    data = {}
+    batch = BatchFactory()
+    record = CIPRSRecordFactory(batch=batch)
+    record.refresh_record_from_data()
```

Comment:
I wonder if we want to use ``refresh_record_from_data``, since it's static in the factory.